packages/amplify-e2e-tests/src/__tests__/__snapshots__/api_10.test.ts.snap

```diff
@@ -15,6 +15,9 @@ $util.unauthorized()
   #set( $currentClaim1 = $util.defaultIfNull($ctx.identity.claims.get(\\"username\\"), $util.defaultIfNull($ctx.identity.claims.get(\\"cognito:username\\"), null)) )
   #if( !$util.isNull($ownerClaim0) && !$util.isNull($currentClaim1) )
     #set( $ownerClaim0 = \\"$ownerClaim0::$currentClaim1\\" )
+    #if( $isAuthorized && $util.isNull($ownerEntity0) && !$ctx.args.input.containsKey(\\"owner\\") )
```

Comment:
Any idea how this snapshot got out of sync?

Reply:
of course, this test never run after the `useChildAccountCredentials` got broken. 
few weeks ago we were wondering why such change was caught by tests in data but not cli... now we know...